src/classes/testDatesToCurrentDate.cls

```diff
@@ -0,0 +1,19 @@
+@isTest
+public class testDatesToCurrentDate {
+    @isTest static void TestStatusDone(){
+        Work_Item__c w = new Work_Item__c(Type__c='Idea',
+                                         Title__c='Test Trigger Done', Assignee__c='0036A00000Ru25rQAB',
```

Comment:
Tests should not be dependent on data that already exists in the org and should not have hard-coded Ids. Please create any new records you will be referencing inside your test method or as part of the setup method.